src/routes/api/bookings/queryBookings.ts

```diff
@@ -0,0 +1,33 @@
+import { Request, Response } from 'express';
+import Booking from '../../../models/booking.model';
+
+/**
+ * Route to query all bookings or the bookings from one booker
+ * @author Kevin Wang
+ */
+const queryBookingsRoute = async (req: Request, res: Response): Promise<void> => {
+    const bookerEmail = req.query.id;
```

Comment:
This lets people arbitrarily view people's bookings - can we authenticate this request?